src/com/mellanox/cicd/Matrix.groovy

```diff
@@ -111,7 +111,7 @@ def forceCleanup(prefix="") {
 
     def cmd = """
     set -eE
-    $prefix bash -c 'shopt -s dotglob; rm -rf ${env.WORKSPACE}/*'
+    $prefix bash -c 'shopt -s dotglob; sudo rm -rf ${env.WORKSPACE}/*'
```

Comment:
it is already like this: https://github.com/Mellanox/ci-demo/blob/master/src/com/mellanox/cicd/Matrix.groovy#L132

see line https://github.com/Mellanox/ci-demo/blob/master/src/com/mellanox/cicd/Matrix.groovy#L132

it tried 1st to remove as jenkins user and if fails - does with sudo.
we can always do with sudo if needed
what issue do you see?